### PR TITLE
Dev 2

### PR DESCRIPTION
**Pull request by ChrisDiep**
- Refactored signup and login to react-bootstrap
- Added functional icon change for show/hide password

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function App() {
     <div>
       <BrowserRouter>
         <div>
-          <div className = 'landing'>
+          <div key={Date.now()} className = 'landing' style={window.location.pathname === '/' ? {height: "100%"} : {}}>
```
Comment (andrewdoanutz) [inline frontend/src/App.js]: Why are we doing this? I deleted the key and styling + line 8-10 in index.css and the background works as intended

**Review by andrewdoanutz — APPROVED** [review]
See comments

**Review by andrewdoanutz — CHANGES_REQUESTED** [review]
see comments

File: frontend/src/App.js (hunk 1 above)
Comment (ChrisDiep) [reply frontend/src/App.js]: That was residual code from before that bug fix, will remove asap

Comment (andrewdoanutz) [reply frontend/src/App.js]: I meant the inline style too. When you delete line 8-10 in index.css and remove the inline styling, everything works. Do you see different on your computer?

Comment (ChrisDiep) [reply frontend/src/App.js]: Oh I'll get on removing that too, that was also residual

**Pull request merged**